common/api/core-backend.api.md

```diff
@@ -4152,21 +4152,47 @@ export class SpatialViewDefinition extends ViewDefinition3d {
     toJSON(): SpatialViewDefinitionProps;
 }
 
+// @public (undocumented)
+export namespace SQLiteDb {
+    // (undocumented)
+    export type CloudContainer = IModelJsNative.CloudContainer;
```

Comment:
We do not publish documentation for types in @bentley/imodeljs-native and until now we have avoided exposing anything from that package -- it's all implicitly `@internal`. Do you intend to change that?

Reply:
No, that's the very problem i'm attempting to solve here. I explicitly "hoist" these types from imodeljs-native into core-backend, just so i *can* make them public (they were previously part of our public api, but exposed from imodeljs-native).

But, they do need documentation here.

Reply:
Exporting the type isn't the same. e.g., you can't do `new SQLiteDb.CloudContainer(props)`.
Where will the documentation for the fields and methods come from?

Reply:
> The documentation for the fields comes from NativeLibrary.ts, nicely.

Are you talking about mousing over stuff in an IDE? I'm talking about our documentation site.

Reply:
btw, you can't do a `new IModelJSNative.CloudContainer` either. You have to do `new IModelHost.platform.CloudContainer`

Reply:
But unlike `new SQLiteDb.CloudContainer`, the compiler accepts `new IModelJsNative.CloudContainer` (maybe that's even worse).

I guess I'm unclear on what you want to make public from IModelJsNative and why. For example, SQLiteDb.nativeDb is now `@public`. Which parts of its API do you expect people to use?

Reply:
I don't want to make any of it public (SQLiteDb.nativeDb becoming public was an accident, thank you.)